spacy/tests/lang/ti/tokenizer.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import unicode_literals
+
+from spacy.lang.ti import Tigrinya
+
+nlp = Tigrinya()
+text = '''"ንአስመራ ንኺድ!"'''
+doc = nlp(text)
+tok_exp = nlp.tokenizer.explain(text)
+assert [t.text for t in doc if not t.is_space] == [t[1] for t in tok_exp]
+for t in tok_exp:
+    print(t[1], "\t", t[0])
+
```

Comment:
Can you reformat this test, by putting the code in a function that starts with `test_` ? In fact, to be consistent, it would be nice to call the file `test_tokenizer.py` instead of `tokenizer.py`. Or just add the file to `test_text.py`. And you probably want to remove the `print` statements ;-)